.circleci/config.yml

```diff
@@ -109,7 +109,6 @@ alias:
     docker:
       - image: cita/cita-build:ubuntu-18.04-20190515
     working_directory: ~/cita-build
-    resource_class: xlarge
```

Comment:
should not remove this line, should comment it out and add a comment for why doing this, for example:

```
# require xlarge class to run all test cases
# resource_class: xlarge
```

Reply:
After transfering cita repo to new originazation,  xlarge is not available on free plan in circle ci.

https://circleci.com/docs/2.0/configuration-reference/#resource_class

Reply:
this is an important info, should not remove it, once we use pain plan in future we can have a clue.